composer.log

```diff
@@ -54,3 +54,6 @@ b4d1a71ae39798e387fe895acb3c924c|Oscar Merida|feature/DIGITAL-140-security-kit|T
 bd7840ebd67d69715dbd07eef9ceb742|cathybaptista|feature/DIGITAL-114-implement-identifier-block|Thu Dec 19 12:41:31 EST 2024|./composer.sh require drupal/config_pages
 72cca00bd0b4403e0db152c489a8e0a1|Matt Poole|feature/DIGITAL-114-implement-identifier-block|Thu Dec 19 15:32:32 EST 2024|./composer.sh require drupal/link_class
 b9a62f1676c64c163727b0feef5db80f|christian medders|feature/DIGITAL-40-gtm|Fri Dec 20 15:46:24 EST 2024|./composer.sh require drupal/google_tag:^2.0
+04623e77b444f74a89e325204132f21e|Ray Estrada|develop|Thu Jan  2 15:57:40 PST 2025|./composer.sh require drupal/role_watchdog
+c6de2b39a10db6f14c638686bb0ca7bf|Ray Estrada|feature/DIGITAL-283-security-modules|Thu Jan  2 16:05:43 PST 2025|./composer.sh require drupal/upgrade_status
+7807a44fd910d0a0d8974b0b285be490|Ray Estrada|feature/DIGITAL-283-security-modules|Thu Jan  2 16:10:28 PST 2025|./composer.sh require drupal/database_ssl_check
```

Comment:
Looks like this will need composer.lock updated.  Any PR that sits more than a couple days usually runs into this issue. :/